bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
This feels like premature optimisation / overengineering to me much like the express parsers and favicon. We don't need a executable to launch this and not sure if we ever will.